.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,60 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main ]
+  schedule:
+    - cron: '17 23 * * 3'
```

```suggestion
    - cron: '17 23 * * 3' # run at 08:17 AM Thursdays, JST
```
The comment would be useful. Could you please confirm the conversion into JST is correct or not?

---

Thank you for suggestion. Time conversion looks correct. Wed 23:17 UTC -> Thu 08:17 JST(+9)